pkg/server/server_test.go

```diff
@@ -38,18 +38,6 @@ var (
 	testKubeConfig = fmt.Sprintf("%s/kubeconfig", testDir)
 )
 
-func TestStringDecode(t *testing.T) {
```

Comment:
Not sure if I missed it but I don't see anything testing `DecodeIgnitionFileContents`. Would it make sense to move this test to `helpers_test.go`?